packages/pretty-reporter/src/format-single-diagnostic.ts

```diff
@@ -1,18 +1,19 @@
 import { relative } from 'node:path'
 import figures from 'figures'
 import terminalLink from 'terminal-link'
-import chalk from 'chalk'
+import picocolors from 'picocolors'
```

Comment:
suggestion (non-blocking): I think the convention is to import is as `pc`, also helps make the code shorter
```suggestion
import pc from 'picocolors'
```

Reply:
Yep, I'll shorten the variable name